db/migrate/20200222165406_add_chef_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddChefToUsers < ActiveRecord::Migration[5.2]
```

Comment:
please change this migration. it should not be called "add chef to users" @Khalidir

Reply:
the name of the migration was changed